js/components/portal-dashboard/class-nav.tsx

```diff
@@ -16,16 +16,21 @@ interface IProps {
   sortByMethod: string;
   studentCount: number;
   trackEvent: (category: string, action: string, label: string) => void;
+  viewMode: string;
 }
 
 export class ClassNav extends React.PureComponent<IProps> {
   render() {
-    const { anonymous, setAnonymous } = this.props;
+    const { anonymous, setAnonymous, viewMode } = this.props;
+    const containerLabel = viewMode === "FeedbackReport" ? "Awaiting feedback" : "Class";
+    const containerLabelType = viewMode === "FeedbackReport" ? undefined : "students";
+    const numItems = viewMode === "FeedbackReport" ? 5 : this.props.studentCount;
+
```

Comment:
I thought the `ClassNav` component is only used on the main portal dashboard (the initial view) and only ever has one mode (where it shows the class name and size and the student sort).  And then we use the `PopupClassNav` inside of the response details and feedback view.  Is any of this new code in this module needed?  I'm confused how we get to a mode where it needs to show something else.  I must be missing something that got added or changed.

Reply:
The hardcoded 5 is also suspicious.  Wondering how this new code gets invoked and what it does.

Reply:
Sorry, I'm pretty sure the changes to `ClassNav` here are at least mostly a relic of me mistaking it for `PopupClassNav.` I'll undo the changes.

Reply:
Ah, got it!  Sounds good!